catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/dyn/DynConstructors.java

```diff
@@ -253,7 +253,7 @@ public <T> Builder hiddenImpl(Class<T> targetClass, Class<?>... types) {
 
       try {
         Constructor<T> hidden = targetClass.getDeclaredConstructor(types);
-        AccessController.doPrivileged(new MakeAccessible(hidden));
+        java.security.AccessController.doPrivileged(new MakeAccessible(hidden));
```

Comment:
`AccessController` is deprecated in JDK17 with no replacement, so currently suppress the warning as a workaround.